packages/scandipwa/src/style/cms/block/homepage-category-preview.scss

```diff
@@ -73,5 +73,9 @@
 
     a {
         display: block;
+        .Button {
```

Comment:
Please use BEM to style elements in task. Block is HomepageCategories and elem is Button. 
So you can make &-Button and it should get your styles.